src/kiota/KiotaHost.cs

```diff
@@ -218,6 +221,26 @@ public static class KiotaHost {
         outputOption.ArgumentHelpName = "path";
         return outputOption;
     }
+    private static Option<List<string>> GetDisableValidationRulesOption() {
+        var parameterName = "--disable-validation-rules";
+        var option = new Option<List<string>>(parameterName, () => new List<string>(), "The OpenAPI description validation rules to disable. Accepts multiple values.");
+        option.AddAlias("--dvr");
+        if(typeof(NoServerEntry).Namespace is string nsName &&
+            Assembly.GetAssembly(typeof(NoServerEntry)) is Assembly assembly) {
+            var validationRules = assembly.GetTypes()
```

Comment:
```suggestion
            var validationRules = typeof(NoServerEntry).Assembly.GetTypes()
```
As they are packaged together, would it make sense to access directly and avoid the `if` statement ?

Reply:
Using if because the APIs are returning nullable references in both cases, so it's to guard from that even though in practice it wouldn't compile.
Trying to avoid using the bang operator in the code base.